action.yaml

```diff
@@ -50,6 +50,7 @@ runs:
     - name: Install Toit packages
       shell: bash
       run: |
+        cd ${{ github.action_path }}
```

Comment:
Can't you use 'working-directory:' for the step instead?

Reply:
yes. That's much better.
https://github.com/toitlang/action-sign-server/pull/14